CDCgov/prime-reportstream
# HL7 SPM Mapping for Outbound OML

This PR  adds mapping to HL7 SPM fields for the outbound OML message type

Test Steps:

1. Run `./gradlew primeCLI --args='fhirdata --output-format HL7 --input-file "./src/testIntegration/resources/datatests/FHIR_to_HL7/sample_OML_20230831-0001.fhir"  -s "./metadata/hl7_mapping/receivers/Flexion/TILabOrder.yml"'`
2. Check that the output HL7 message has the SPM field

## Changes
- created `specimen.yml` schema to be consumed by the `OML_O21.yml` base

## Checklist

### Testing

## Linked Issues
[Create Outbound OML SPM Segment](https://app.zenhub.com/workspaces/cdc-ti-633de70d939f6c00155f3722/issues/gh/cdcgov/trusted-intermediary/494)
